src/main/webapp/app/exercises/programming/hestia/git-diff-report/git-diff-file-panel.component.ts

```
@@ -2,57 +1,66 @@
-        this.previousFilePath = this.diffEntries
-            .map((entry) => entry.previousFilePath)
-            .filter((filePath) => filePath)
-            .first();
+    readonly addedLineCount = computed(
+        () =>
+            this.diffEntries()
+                .filter((entry) => entry && entry.filePath && entry.startLine && entry.lineCount)
+                .flatMap((entry) => {
+                    return this.modifiedFileContent()
+                        ?.split('\n')
+                        .slice(entry.startLine! - 1, entry.startLine! + entry.lineCount! - 1);
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)